corehq/ex-submodules/couchforms/tests/data/edit.json

```diff
@@ -1,79 +0,0 @@
-{
-    "_attachments": {
-        "form.xml": {
-            "content_type": "text/xml",
-            "digest": "md5-eXs062nHm8phI6BSm6hTcg==",
-            "length": 1195,
-            "revpos": 1,
-            "stub": true
-        }
-    },
-    "_id": "7H46J37FGH3",
-    "_rev": "1-7c5cf73b344dfcadbb169c85a03c3635",
-    "computed_": {},
-    "computed_modified_on_": null,
-    "doc_type": "XFormInstance",
-    "form": {
-        "#type": "general",
-        "@xmlns": "http://openrosa.org/app/general",
-        "Meta": {
-            "TimeEnd": "2010-07-24T13:55:11Z",
-            "TimeStart": "2010-07-22T13:54:27Z",
-            "clinic_id": "5020280",
-            "uid": "7H46J37FGH3",
-            "user_id": "f7f0c79e-8b79-11df-b7de-005056c00008",
-            "username": "admin"
-        },
-        "assessment": {
-            "categories": "Edited Baby!"
-        },
-        "case": {
-            "case_type": "malaria",
-            "followup_date": "",
-            "followup_type": "referral",
-            "outcome": "",
-            "patient_id": "6713c1c211b3af6832c8ad781e310d98"
-        },
-        "chw_referral_id": "",
-        "danger_signs": "convuls",
-        "diagnosis": "malaria",
-        "drugs": {
-            "dispensed": "",
-            "prescribed": ""
-        },
-        "encounter_date": "2010-07-22",
-        "encounter_type": "new_case",
-        "general_exam": "none",
-        "hiv_result": "r",
-        "investigations": {
-            "categories": ""
-        },
-        "medical_history": "daibetes",
-        "on_haart": "",
-        "phys_exam_detail": "none",
-        "resolution": "referral",
-        "vitals": {
-            "bp": "",
-            "heart_rate": "",
-            "height": "100",
-            "resp_rate": "",
-            "temp": "",
-            "weight": ""
-        }
-    },
-    "history": [],
-    "initial_processing_complete": false,
-    "partial_submission": false,
-    "received_on": "2013-09-21T04:14:10Z",
-    "xmlns": "http://openrosa.org/app/general",
-    "#export_tag": null,
-    "app_id": null,
-    "auth_context": {},
-    "build_id": null,
-    "date_header": null,
-    "domain": null,
-    "last_sync_token": null,
-    "openrosa_headers": {},
-    "path": null,
-    "submit_ip": null
-}
```

Comment:
I'm confused then, because this commit doesn't modify `PostTest` not to reference this file. Doesn't it mean removing this will make that test fail?